ArchiSteamFarm.OfficialPlugins.Monitoring/MonitoringPlugin.cs

```diff
@@ -98,6 +108,33 @@ internal sealed class MonitoringPlugin : OfficialPlugin, IWebServiceProvider, IG
 
 	public override Task OnLoaded() => Task.CompletedTask;
 
+	private static Measurement<int> CreateRuntimeInformation() {
+		string framework = RuntimeInformation.FrameworkDescription.Trim();
+
+		if (framework.Length == 0) {
+			framework = "unknown";
```

Comment:
I'm pretty sure we have similar code already somewhere else, extract it to a public function and use it.

Reply:
Yes. That's copied 1:1 from OS.cs lines 55-83; I wasn't sure you'd appreciate every single one of them being a public property. Will do.

Reply:
However, there is one difference: In monitoring applications "unknown" is much better to filter for than "Unknown Framework", "Unknown Runtime" and "Unknown OS".